srcpkgs/looking-glass/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'looking-glass'
+pkgname=looking-glass
+version=5.0.1
+revision=1
+archs="x86_64"
+wrksrc="$pkgname-B$version/client"
+build_style=cmake
+configure_args="-DENABLE_BACKTRACE=no"
+makedepends="cmake make gcc libglapi libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel pkg-config libX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp libwaylandpp-devel wayland-protocols dejavu-fonts-ttf libXScrnSaver-devel MesaLib-devel"
+depends="dejavu-fonts-ttf"
+short_desc="Extremely low latency KVMFR for guests with VGA PCI Passthrough"
+maintainer="lay-by <bean_s@riseup.net>"
+license="GPL-2.0-or-later"
+homepage="https://looking-glass.io/"
+distfiles="$homepage/artifact/B$version/source>$pkgname-B$version.tar.gz"
```

Comment:
```suggestion
distfiles="https://looking-glass.io/artifact/B$version/source>$pkgname-$version.tar.gz"
```

Reply:
I changed all of the parts you mentioned, thanks for your help and patience.